### PR TITLE
Update index.markdown

### PR DESCRIPTION
**Pull request by blcham**

### CODE REVIEW

File: _pages/projects.markdown [hunk 1]
```diff
@@ -23,8 +23,6 @@ and technologies in the aforementioned information systems. The topic of our res
 - Semantic Web-based information systems and software libraries
 - Ontology-based information extraction from text
 
- **TODO: add newer projects, e.g. IPR**
```
[the remark this answers is not in the document]
Comment (blcham) [reply _pages/projects.markdown:26]: Yes, I we created ticket for that, but we can publish to production without complete-ing the ticket.

**Comment by blcham** [comment]
I have removed all the TODOs from all pages and moved it to tickets: #37, #38, #39.

File: _pages/projects.markdown (hunk 1 above)
[the remark this answers is not in the document]
Comment (blcham) [reply _pages/projects.markdown:26]: resolving this.

File: _pages/projects.markdown [hunk 2]
```diff
@@ -23,68 +23,63 @@ and technologies in the aforementioned information systems. The topic of our res
...
-
 ## Semantic Vocabulary Management Tool for Urban Planning Data (OPPPR FEL ICT)
 
-The project lasted from 1st January 2018 to the 30th June 2020 and dealt with the development of a vocabulary and the semantic model in the Urban Planning domain. Project partner was Prague Institute of Planning and Development. During the project was used and further developed a Semantic Vocabulary Management Tool called Termit. In the project it was used exclusively for development of ontologies describing Urban Planning Data.
+The project dealt with the development of a vocabulary and the semantic model in the Urban Planning domain. Project partner was Prague Institute of Planning and Development. During the project was used and further developed a Semantic Vocabulary Management Tool called Termit. In the project it was used exclusively for development of ontologies describing Urban Planning Data.
```
Comment (blcham) [inline _pages/projects.markdown:28]:
```suggestion
The project dealt with the development of a vocabulary and the semantic model in the Urban Planning domain. Project partner was Prague Institute of Planning and Development. During the project, a Semantic Vocabulary Management Tool called Termit was used and further developed . In the project it was used exclusively for development of ontologies describing Urban Planning Data.
```

**Review by blcham — COMMENTED** [review]
LGTM, i cannot approved because I created the ticket, can be merged.

**Review by MichalMed — APPROVED** [review]

**Pull request closed without merging**